R/chooseFromList.R

```diff
@@ -121,7 +121,9 @@ chooseFromList <- function(theList, type = "items", userinfo = NULL, addAllPatte
     }
   }
   booleanList[identifier] <- TRUE
-  message("Selected: ", paste(originalList[identifier], collapse = ", "))
+  msgselected <- originalList[identifier]
+  stopafter <- min(which(cumsum(nchar(msgselected)) > 666), length(msgselected))
+  message("Selected: ", paste(msgselected[1:stopafter], collapse = ", "), if (stopafter < length(msgselected)) ", ...")
```

Comment:
```suggestion
  message("Selected: ", paste(msgselected[seq_len(stopafter)], collapse = ", "), if (stopafter < length(msgselected)) ", ...")
```
This handles the case stopafter = 0 better. Not sure if you catch that earlier, but doing it this way never hurts (I think the linter also warns and suggests this)